README.md

```diff
@@ -7,3 +7,324 @@ Copyright 2019, Yung-Yu Chen <yyc@solvcon.net>.  All rights reserved.
+* There will be 6 homework assignments for you to exercise.  Programming in Python and/or C++ is required.
+* Mid-term examination will be conducted to assess students' understandings to the analytical materials.
+* Term project will be used to assess students' overall coding skills.  Presentation is required.  Failure to present results in **0 point** for this part.  Check [the term project page](term_project.ipynb) before you start.
+
+# How to study
+
+* This is a practical course.  No textbook is available for this specific interdisciplinary subject.
+* To study the subject, students are required to research with online documents and source code, and write programs to practice.
+* In-class instruction and [course notes](https://github.com/yungyuc/nsd) are provided for guidance.
+* References:
+  * Computer Systems: A Programmer's Perspective, Randal E. Bryant and David R. O'Hallaron: https://csapp.cs.cmu.edu/
+  * Python documentation: https://docs.python.org/3/
+  * Cppreference: https://en.cppreference.com/
+  * [Effective Modern C++](https://www.oreilly.com/library/view/effective-modern-c/9781491908419/), Scott Meyer, O'Reilly, 2014
+  * Source code: [cpython](https://github.com/python/cpython), [numpy](https://github.com/numpy/numpy), [xtensor](https://github.com/QuantStack/xtensor), and [pybind11](https://github.com/pybind/pybind11)
+
+# Course schedule
+
+* W1 (9/14) [Lecture 1](#Lecture-1-Introduction): Introduction
+* W2 (9/21) [Lecture 2](#Lecture-2-Fundamental-engineering-practices): Fundamental engineering practices (*homework \#1*)
+* W3 (9/28) [Lecture 3](#Lecture-3-Python-and-numpy): Python and numpy (*term project proposal start*)
+* W4 (10/5) [Lecture 4](#Lecture-4-C++-and-computer-architecture): C++ and computer architecture (*homework \#2*)
+* W5 (10/12) [Lecture 5](#Lecture-5-Matrix-operations): Matrix operations
+* W6 (10/19) [Lecture 6](#Lecture-6-Cache-optimization): Cache optimization (*homework \#3*)
+* W7 (10/26) [Lecture 7](#Lecture-7-SIMD): SIMD (*term project proposal due*)
+* W8 (11/2) Mid-term examination (term project proposal submission due on 4/29)
+* W9 (11/9) [Lecture 8](#Lecture-8-Memory-management): Memory management (*homework \#4*)
+* W10 (11/16) [Lecture 9](#Lecture-9-Smart-pointers): Ownership and smart pointers
+* W11 (11/23) [Lecture 10](#Lecture-10-Modern-C++): Modern C++ (*homework \#5*)
+* W12 (11/30) [Lecture 11](#Lecture-11-C++-and-C-for-Python): C++ and C for Python
+* W13 (12/7) [Lecture 12](#Lecture-12-Array-code-in-C++): Array code in C++ (*homework \#6*)
+* W14 (12/14) [Lecture 13](#Lecture-13-Array-oriented-design): Array-oriented design
+* W15 (12/21) [Lecture 14](#Lecture-14-Advanced-Python): Advanced Python
+* W16 (12/28) Term project presentation
+* W17 (1/4) No meeting (optional lecture is not planned)
+* W18 (1/11) No meeting (optional lecture is not planned)
+
+# Lecture 1 [Introduction](01_introduction/introduction.ipynb)
```

Comment:
The link link doesn't seem to work?  I think the target should point to `notebook/20au_nctu/01_introduction/01_introduction.ipynb`.

If external links don't work in the GitHub page, we should remove them.